src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -752,15 +753,15 @@ export const terminalContributionsDescriptor: IExtensionPointDescriptor<ITermina
 					}],
 					properties: {
 						id: {
-							description: nls.localize('vscode.extension.contributes.terminal.quickFixes.id', "The ID of the quick fix."),
+							description: nls.localize('vscode.extension.contributes.terminal.quickFixes.id', "The ID of the quick fix provider."),
 							type: 'string',
 						},
 						commandLineMatcher: {
-							description: nls.localize('vscode.extension.contributes.terminal.quickFixes.commandLineMatcher', "The command line to match."),
+							description: nls.localize('vscode.extension.contributes.terminal.quickFixes.commandLineMatcher', "The regular expression to test the command line against."),
 							type: 'string',
 						},
 						outputMatcher: {
-							description: nls.localize('vscode.extension.contributes.terminal.quickFixes.outputMatcher', "The output to match, which provides groups of the form <group_name> to be referenced via ${group:group_name} in commandToRun and linkToOpen."),
+							description: nls.localize('vscode.extension.contributes.terminal.quickFixes.outputMatcher', "The regular expression to test the output against, which provides groups of the form <group_name> to be referenced via ${group:group_name} in terminalCommand and uri."),
```

Comment:
Do any contributions like this have examples in them? It's hard to understand this without one.

Reply:
we should probably only do that when the API has been finalized though